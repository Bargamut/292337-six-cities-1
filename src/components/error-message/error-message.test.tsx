import * as React from 'react';
import * as renderer from 'react-test-renderer';

import ErrorMessage from './error-message';

it(`Error Mesage renders correctly`, () => {
  const errorMessage = renderer.create(<ErrorMessage error="Ошибка!" />);

  expect(errorMessage).toMatchSnapshot();
});
