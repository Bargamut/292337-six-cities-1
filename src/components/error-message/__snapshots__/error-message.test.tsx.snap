// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Mesage renders correctly 1`] = `
<span
  className="error error__validation"
>
  Ошибка!
</span>
`;
